recipes/agat/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "agat" %}
+{% set version = "0.0.1" %}
+{% set md5 = "ce31a9c4e1a8ba4a79bfbe79a2c18684" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/NBISweden/AGAT/archive/v{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  host:
+    - perl >=5.8
+    - perl-file-sharedir-install
+    - perl-file-share
+    - perl-bioperl
+    - perl-clone
+    - perl-graph
+    - perl-lwp-simple
+    - perl-statistics-r
+    - perl-json
+    - perl-sort-naturally
+    - libdb
+  run:
+    - perl >=5.8
+    - perl-bioperl
+    - perl-file-share
+    - perl-clone
+    - perl-graph
+    - perl-lwp-simple
+    - perl-statistics-r
+    - perl-json
+    - perl-sort-naturally
+    - r-base
+    - libdb
+
+about:
+  home: https://github.com/NBISweden/AGAT
+  license: GPLv3
+  license_family: GPL
+  license_file: LICENSE
+  summary: Another Gff Analysis Toolkit (AGAT). Suite of tools to handle gene annotations in any GTF/GFF format.
+  dev_url: https://github.com/NBISweden/AGAT
+
+extra:
+  recipe-maintainers:
+    - juke34
+  skip-lints:
+    - missing_tests
```

Comment:
Can you add some tests? There appear to be scripts that one could run, maybe checking the output of `--help` for one of them.

Reply:
I tried but it didn't work...
I had 

```
test:
     commands:
        - agat_sp_gxf_to_gff3.pl -h
```

but it was crashing at the singularity check (`Can't locate Bio/Tools/GFF.pm`).... telling that Bioperl module was missing while it is installed (`bioconda/linux-64::perl-bioperl-1.6.924-4`). Could it be related to Bioperl version?:

```
.22:01:13 BIOCONDA INFO (ERR) [Nov 20 22:01:13] INFO Invoke Task [singularity]
22:01:13 BIOCONDA INFO (ERR) [Nov 20 22:01:13] WARN Invoked Stepless Task [singularity]
22:01:13 BIOCONDA INFO (ERR) [Nov 20 22:01:13] INFO Invoke Task [cleanup]
22:01:13 BIOCONDA INFO (ERR) [Nov 20 22:01:13] STEP Run image [quay.io/dpryan79/mulled_container:latest] with command [[rm -rf /data/dist]]
22:01:13 BIOCONDA INFO (ERR) [Nov 20 22:01:13] DEBU Creating container [step-ea14abe96d]
22:01:13 BIOCONDA INFO (ERR) [Nov 20 22:01:13] DEBU Created container [b567ae4fba44 step-ea14abe96d], starting it
22:01:13 BIOCONDA INFO (ERR) [Nov 20 22:01:13] DEBU Container [b567ae4fba44 step-ea14abe96d] started, waiting for completion
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] DEBU Container [b567ae4fba44 step-ea14abe96d] completed with exit code [0] as expected
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] DEBU Container [b567ae4fba44 step-ea14abe96d] removed
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] INFO Invoke Task [test]
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] STEP Run image [quay.io/biocontainers/agat:0.0.1--pl526r35_0] with command [[agat_sp_gxf_to_gff3.pl -h]]
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] DEBU Creating container [step-a1d1d5a17b]
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] DEBU Created container [8f77470d14df step-a1d1d5a17b], starting it
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] DEBU Container [8f77470d14df step-a1d1d5a17b] started, waiting for completion
22:01:14 BIOCONDA INFO (ERR) [Nov 20 22:01:14] SERR Can't locate Bio/Tools/GFF.pm in @INC (you may need to install the Bio::Tools::GFF module) (@INC contains: /usr/local/lib/site_perl/5.26.2/x86_64-linux-thread-multi /usr/local/lib/site_perl/5.26.2 /usr/local/lib/5.26.2/x86_64-linux-thread-multi /usr/local/lib/5.26.2 .) at /usr/local/lib/site_perl/5.26.2/AGAT/OmniscientI.pm line 10.
```

But le perl package contains tests that are run during installation so I though it was useless to add a test:
```
.08:10:21 BIOCONDA INFO (OUT) t/gff_syntax.t ... ok
08:10:22 BIOCONDA INFO (OUT) t/gff_version.t .. ok
```